### PR TITLE
Fixed typos in count-backwards-with-a-for-loop.md

### LINKED ISSUE #40470: there is a spelling error in this lesson "Count Backwards With a For Loop"
**Describe your problem and how to reproduce it:**
here in this lesson we this explanation: In order to count backwards by twos, we'll need to change our initialization, condition, and final-expression.
 backwards should change into backward and twos should change into two. 

**Add a Link to the page with the problem:**

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/count-backwards-with-a-for-loop

**Tell us about your browser and operating system:**
* Browser Name: chrome 
* Browser Version: 
* Operating System: Windows 10

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

### PR DESCRIPTION
**Pull request by matias-miller**
Checklist:

- [ X ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ X ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ X ] My pull request targets the `master` branch of freeCodeCamp.
- [ X ] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40470

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/count-backwards-with-a-for-loop.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ forumTopicId: 16808
 
 A for loop can also count backwards, so long as we can define the right conditions.
 
-In order to count backwards by twos, we'll need to change our `initialization`, `condition`, and `final-expression`.
+In order to count backward by two, we'll need to change our `initialization`, `condition`, and `final-expression`.
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/count-backwards-with-a-for-loop.md:13]: What about the following?
```suggestion
In order to count back by two each time, we'll need to change our `initialization`, `condition`, and `final-expression`.
```

**Comment by ieahleen** [comment]
Do not make changes in your master branch, you should really make a branch on which you make the changes

step 3 here in "making changes locally" of the contributing docs

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/count-backwards-with-a-for-loop.md (hunk 1 above)
Comment (ShaunSHamilton) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/count-backwards-with-a-for-loop.md:13]: Would it be better to use more familiar jargon:
> In order to decrement by two each iteration...

Comment (RandellDawson) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/count-backwards-with-a-for-loop.md:13]: @matias-miller I have updated my suggestion to reflect @ShaunSHamilton's wording.  Please click the `Commit suggestion` button above and write an appropriate commit message for it.  Thanks.

**Comment by RandellDawson** [comment]
@matias-miller You have not made the changes suggested by @ShaunSHamilton.  Can you please commit his suggested changes?  Thank you.

**Review by RandellDawson — APPROVED** [review]
LGTM

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me.

**Pull request merged**